docs/contributing/API Review Process.md

```diff
@@ -31,7 +31,7 @@ The rule of thumb is that we (**dotnet/roslyn**) review every public API that is
 
  There are three methods to get an API review (this section applies to API area owners, but is included for transparency into the process):
 
-* **Get into the backlog**. Generally speaking, filing an issue in `dotnet/roslyn` and applying the label `api-ready-for-review` on it will make your issue show up during API reviews. The downside is that we generally walk the backlog oldest-newest, so your issue might not be looked at for a while. Progress of issues can be tracked via https://apireview.net/backlog/roslyn (PROTOTYPE: make this link actually work).
+* **Get into the backlog**. Generally speaking, filing an issue in `dotnet/roslyn` and applying the label `api-ready-for-review` on it will make your issue show up during API reviews. The downside is that we generally walk the backlog oldest-newest, so your issue might not be looked at for a while. Progress of issues can be tracked via https://apireview.net/backlog?g=Roslyn.
```

Comment:
See <https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md034---bare-url-used>

> Rationale: Without angle brackets, the URL isn't converted into a link in many markdown parsers.


```suggestion
* **Get into the backlog**. Generally speaking, filing an issue in `dotnet/roslyn` and applying the label `api-ready-for-review` on it will make your issue show up during API reviews. The downside is that we generally walk the backlog oldest-newest, so your issue might not be looked at for a while. Progress of issues can be tracked via <https://apireview.net/backlog?g=Roslyn>.
```

Reply:
The lack of actual standardization in markdown is disappointing to say the least.

Reply:
Using markdownlint helps a little bit. But I'm not sure if GH Actions are supported in the repository.